server/app/views/v1/grid_stats/stats.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+json.from @node_stats[:from_time]
+json.to @node_stats[:to_time]
+json.stats @node_stats[:stats] do |stat|
+  json.timestamp stat[:timestamp]
+  json.data_points stat[:data_points]
```

Comment:
Just the number of data points in this stat.  It's probably not helpful for the front end, I mostly had it in there for the sake of unit tests.  I'll take it out.